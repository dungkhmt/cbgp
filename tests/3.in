100 100
100 208
13 4
14 5
82 10
55 68
40 38
82 74
20 50
5 65
80 20
8 73
69 5
37 9
14 29
3 6
24 62
95 12
11 65
4 1
91 8
7 24
4 2
37 72
93 73
78 71
7 8
26 85
33 60
6 2
66 32
80 85
75 17
72 69
73 91
17 94
78 6
3 65
66 69
5 100
32 5
4 15
26 80
8 77
43 48
11 64
13 92
21 4
33 80
35 32
4 27
37 2
26 20
60 26
5 66
36 65
39 31
20 22
54 70
88 89
59 14
5 2
1 15
1 33
36 40
93 8
100 72
76 55
54 81
22 1
16 25
40 49
39 30
2 3
7 4
14 17
81 94
81 75
6 56
51 58
11 34
18 17
29 59
70 81
55 32
10 4
14 18
69 2
23 47
20 1
52 98
36 38
33 58
43 13
45 21
36 5
14 81
34 54
18 28
84 4
81 17
5 25
22 32
50 22
71 2
88 58
54 75
9 72
51 10
63 69
63 99
24 67
83 18
20 32
31 25
71 1
31 16
97 3
12 13
6 96
67 86
96 56
51 88
76 68
92 21
18 19
19 47
54 57
10 74
75 14
14 34
2 96
83 14
33 26
4 58
4 12
60 80
43 21
5 61
4 89
13 95
17 29
13 21
6 97
14 90
5 38
39 5
3 96
14 54
20 35
22 71
41 36
14 11
69 87
79 30
81 29
4 33
14 36
79 25
44 52
75 64
3 78
2 9
32 61
78 2
100 69
63 24
64 54
51 89
78 36
21 27
5 9
15 2
30 36
71 32
77 73
32 78
4 69
36 11
35 44
16 5
24 8
78 97
35 46
20 33
36 76
36 42
84 45
92 12
66 61
79 31
4 24
2 65
32 46
51 4
90 54
34 64
3 36
21 12
63 46
2 1
11 5
43 92
4 88
23 19
30 5
25 30
2 72
53 51
56 2
