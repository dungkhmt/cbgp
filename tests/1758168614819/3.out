(12, 3), (13, 4), (81, 9), (54, 67), (39, 37), (81, 73), (19, 49), (4, 64), (79, 19), (7, 72), (68, 4), (36, 8), (13, 28), (2, 5), (23, 61), (94, 11), (10, 64), (3, 0), (90, 7), (6, 23), (3, 1), (36, 71), (92, 72), (77, 70), (6, 7), (25, 84), (32, 59), (5, 1), (65, 31), (79, 84), (74, 16), (71, 68), (72, 90), (16, 93), (77, 5), (2, 64), (65, 68), (4, 99), (31, 4), (3, 14), (25, 79), (7, 76), (42, 47), (10, 63), (12, 91), (20, 3), (32, 79), (34, 31), (3, 26), (36, 1), (25, 19), (59, 25), (4, 65), (35, 64), (38, 30), (19, 21), (53, 69), (87, 88), (58, 13), (4, 1), (0, 14), (0, 32), (35, 39), (92, 7), (99, 71), (75, 54), (53, 80), (21, 0), (15, 24), (39, 48), (38, 29), (1, 2), (6, 3), (13, 16), (80, 93), (80, 74), (5, 55), (50, 57), (10, 33), (17, 16), (28, 58), (69, 80), (54, 31), (9, 3), (13, 17), (68, 1), (22, 46), (19, 0), (51, 97), (35, 37), (32, 57), (42, 12), (44, 20), (35, 4), (13, 80), (33, 53), (17, 27), (83, 3), (80, 16), (4, 24), (21, 31), (49, 21), (70, 1), (87, 57), (53, 74), (8, 71), (50, 9), (62, 68), (62, 98), (23, 66), (82, 17), (19, 31), (30, 24), (70, 0), (30, 15), (96, 2), (11, 12), (5, 95), (66, 85), (95, 55), (50, 87), (75, 67), (91, 20), (17, 18), (18, 46), (53, 56), (9, 73), (74, 13), (13, 33), (1, 95), (82, 13), (32, 25), (3, 57), (3, 11), (59, 79), (42, 20), (4, 60), (3, 88), (12, 94), (16, 28), (12, 20), (5, 96), (13, 89), (4, 37), (38, 4), (2, 95), (13, 53), (19, 34), (21, 70), (40, 35), (13, 10), (68, 86), (78, 29), (80, 28), (3, 32), (13, 35), (78, 24), (43, 51), (74, 63), (2, 77), (1, 8), (31, 60), (77, 1), (99, 68), (62, 23), (63, 53), (50, 88), (77, 35), (20, 26), (4, 8), (14, 1), (29, 35), (70, 31), (76, 72), (31, 77), (3, 68), (35, 10), (34, 43), (15, 4), (23, 7), (77, 96), (34, 45), (19, 32), (35, 75), (35, 41), (83, 44), (91, 11), (65, 60), (78, 30), (3, 23), (1, 64), (31, 45), (50, 3), (89, 53), (33, 63), (2, 35), (20, 11), (62, 45), (1, 0), (10, 4), (42, 91), (3, 87), (22, 18), (29, 4), (24, 29), (1, 71), (52, 50), (55, 1), 
numberIntersection=0 
minAngle=0.01763294682715788 
sumAngle=100.0014712786093 
minEdgeLength=1.0 
sumEdgeLength=11566.430456543943 
minDistance=0.03624997167972069 
sumDistance=2456689.330879214 

0.0000000000 0.0176329468 100.0014712786 1.0000000000 11566.4304565439 0.0362499717 2456689.3308792140
0: (61, 38),
1: (183, 43),
2: (309, 50),
3: (66, 53),
4: (234, 222),
5: (307, 41),
6: (115, 69),
7: (128, 53),
8: (166, 74),
9: (57, 71),
10: (197, 66),
11: (104, 105),
12: (86, 75),
13: (229, 142),
14: (68, 49),
15: (245, 170),
16: (226, 95),
17: (215, 88),
18: (219, 105),
19: (4, 20),
20: (114, 74),
21: (142, 23),
22: (226, 133),
23: (108, 113),
24: (236, 156),
25: (34, 82),
26: (77, 57),
27: (218, 105),
28: (230, 93),
29: (258, 164),
30: (249, 169),
31: (335, 19),
32: (42, 58),
33: (265, 68),
34: (1, 4),
35: (307, 62),
36: (163, 61),
37: (232, 142),
38: (251, 170),
39: (235, 143),
40: (237, 155),
41: (239, 157),
42: (100, 76),
43: (3, 3),
44: (87, 63),
45: (348, 11),
46: (216, 106),
47: (100, 75),
48: (237, 156),
49: (69, 23),
50: (46, 82),
51: (5, 2),
52: (123, 400),
53: (252, 85),
54: (334, 20),
55: (302, 44),
56: (230, 136),
57: (45, 61),
58: (229, 95),
59: (37, 60),
60: (237, 225),
61: (84, 176),
62: (235, 247),
63: (224, 75),
64: (193, 64),
65: (236, 237),
66: (68, 150),
67: (320, 36),
68: (147, 52),
69: (242, 87),
70: (202, 28),
71: (159, 60),
72: (146, 76),
73: (56, 123),
74: (204, 79),
75: (300, 89),
76: (134, 62),
77: (315, 38),
78: (250, 165),
79: (32, 56),
80: (234, 89),
81: (71, 185),
82: (215, 104),
83: (80, 60),
84: (31, 59),
85: (67, 151),
86: (129, 54),
87: (48, 63),
88: (51, 65),
89: (245, 105),
90: (134, 60),
91: (100, 77),
92: (133, 60),
93: (229, 93),
94: (89, 81),
95: (303, 47),
96: (309, 42),
97: (9, 0),
98: (90, 183),
99: (156, 62),
61 38 183 43 309 50 66 53 234 222 307 41 115 69 128 53 166 74 57 71 197 66 104 105 86 75 229 142 68 49 245 170 226 95 215 88 219 105 4 20 114 74 142 23 226 133 108 113 236 156 34 82 77 57 218 105 230 93 258 164 249 169 335 19 42 58 265 68 1 4 307 62 163 61 232 142 251 170 235 143 237 155 239 157 100 76 3 3 87 63 348 11 216 106 100 75 237 156 69 23 46 82 5 2 123 400 252 85 334 20 302 44 230 136 45 61 229 95 37 60 237 225 84 176 235 247 224 75 193 64 236 237 68 150 320 36 147 52 242 87 202 28 159 60 146 76 56 123 204 79 300 89 134 62 315 38 250 165 32 56 234 89 71 185 215 104 80 60 31 59 67 151 129 54 48 63 51 65 245 105 134 60 100 77 133 60 229 93 89 81 303 47 309 42 9 0 90 183 156 62 
0: (61, 38), 1: (183, 43), 2: (309, 50), 3: (66, 53), 4: (234, 222), 5: (307, 41), 6: (115, 69), 7: (128, 53), 8: (166, 74), 9: (57, 71), 10: (197, 66), 11: (104, 105), 12: (86, 75), 13: (229, 142), 14: (68, 49), 15: (245, 170), 16: (226, 95), 17: (215, 88), 18: (219, 105), 19: (4, 20), 20: (114, 74), 21: (142, 23), 22: (226, 133), 23: (108, 113), 24: (236, 156), 25: (34, 82), 26: (77, 57), 27: (218, 105), 28: (230, 93), 29: (258, 164), 30: (249, 169), 31: (335, 19), 32: (42, 58), 33: (265, 68), 34: (1, 4), 35: (307, 62), 36: (163, 61), 37: (232, 142), 38: (251, 170), 39: (235, 143), 40: (237, 155), 41: (239, 157), 42: (100, 76), 43: (3, 3), 44: (87, 63), 45: (348, 11), 46: (216, 106), 47: (100, 75), 48: (237, 156), 49: (69, 23), 50: (46, 82), 51: (5, 2), 52: (123, 400), 53: (252, 85), 54: (334, 20), 55: (302, 44), 56: (230, 136), 57: (45, 61), 58: (229, 95), 59: (37, 60), 60: (237, 225), 61: (84, 176), 62: (235, 247), 63: (224, 75), 64: (193, 64), 65: (236, 237), 66: (68, 150), 67: (320, 36), 68: (147, 52), 69: (242, 87), 70: (202, 28), 71: (159, 60), 72: (146, 76), 73: (56, 123), 74: (204, 79), 75: (300, 89), 76: (134, 62), 77: (315, 38), 78: (250, 165), 79: (32, 56), 80: (234, 89), 81: (71, 185), 82: (215, 104), 83: (80, 60), 84: (31, 59), 85: (67, 151), 86: (129, 54), 87: (48, 63), 88: (51, 65), 89: (245, 105), 90: (134, 60), 91: (100, 77), 92: (133, 60), 93: (229, 93), 94: (89, 81), 95: (303, 47), 96: (309, 42), 97: (9, 0), 98: (90, 183), 99: (156, 62), 
numberIntersection=0 minAngle=0.01763294682715788 sumAngle=100.00147127859834 minEdgeLength=1.0 sumEdgeLength=11566.430456543918 minDistance=0.03624997167972069 sumDistance=2456689.330879308 
Time taken: 43.010 s
